sonar-db/src/main/java/org/sonar/db/permission/PermissionDao.java

```diff
@@ -48,20 +50,37 @@ public PermissionDao(MyBatis myBatis) {
 
   /**
    * @return a paginated list of users.
+   * @deprecated use {@link #selectUserPermissionsByQuery(DbSession, PermissionQuery)} instead
    */
-  public List<UserWithPermissionDto> selectUsers(DbSession session, PermissionQuery query, @Nullable Long componentId, int offset, int limit) {
+  @Deprecated
+  public List<UserWithPermissionDto> selectUsers(DbSession session, OldPermissionQuery query, @Nullable Long componentId, int offset, int limit) {
     Map<String, Object> params = usersParameters(query, componentId);
 
     return mapper(session).selectUsers(params, new RowBounds(offset, limit));
   }
 
-  public int countUsers(DbSession session, PermissionQuery query, @Nullable Long componentId) {
+  /**
+   * Ordered by user names
+   */
+  public List<String> selectLoginsByPermissionQuery(DbSession dbSession, PermissionQuery query) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this method return value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)